### PR TITLE
New InputFile component

### PR DESCRIPTION
**Pull request by builtbyedgar**
Shortcut: https://app.shortcut.com/cartoteam/story/145206/component-input-file

### CODE REVIEW

File: packages/react-ui/src/custom-components/InputFile.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    setOver(false);
+  };
+
+  const handleDrop = (event) => {
+    event.preventDefault();
+
+    const newFiles = [];
+    const items = event.dataTransfer.items;
+
+    for (let i = 0; i < items.length; i++) {
```
Comment (Josmorsot) [inline packages/react-ui/src/custom-components/InputFile.js:74]: Why are you not using a forEach instead?

I would change it for:

```ts
items.forEach(item => { .... })
```

Also I think that would be better to move this block to a new function, something like that:
```ts
// handleDrop function
const newFiles = getAllFiles(items)
props.onChange(newFiles)
```

**Review by Josmorsot — CHANGES_REQUESTED** [review]
Try to use forEach instead.

LGTM, just minor suggestions

**Comment by VictorVelarde** [comment]
Same here @builtbyedgar. Tell us if you'll be able to have availability to fix comments soon or not, please

**Comment by VictorVelarde** [comment]
@Josmorsot. You should please take also this one, when you can

File: packages/react-ui/src/custom-components/InputFile.js (hunk 1 above)
Comment (Josmorsot) [reply packages/react-ui/src/custom-components/InputFile.js:74]: `items` is not an array, so instead using `forEach` I'm going to use a `for(item of items)`

**Review by Josmorsot — APPROVED** [review]

**Pull request merged**